src/components/screens/management-screen.tsx

```diff
@@ -0,0 +1,438 @@
+  };
+
+  /**
+   * Fetches the person data & person total time data
+   */
+  const fetchData = async () => {
+    const fetchedPersons: PersonWithTotalTime[] = [];
+
+    setIsLoading(true);
+
+    try {
+      await Api.getTimeBankApi()
+        .timebankControllerGetPersons()
+        .then(persons => {
+          persons.forEach(person =>
+            fetchedPersons.push({
+              person: person
+            } as PersonWithTotalTime));
+        });
+    } catch (error) {
+      context.setError(strings.errorHandling.fetchUserDataFailed, error);
+    }
+
+    const personsTotalTimePromises = fetchedPersons.map(populatePersonTotalTimeData);
+
+    const fetchedPersonsTotalTime = await Promise.all(personsTotalTimePromises);
+
+    setPersonsTotalTime(fetchedPersonsTotalTime);
+    setDisplayedPersonsTotalTime(fetchedPersonsTotalTime);
+    setIsLoading(false);
+  };
+
+  React.useEffect(() => {
+    if (!accessToken) {
+      return;
+    }
+
+    const decodedToken: any = jwt_decode(accessToken.access_token);
+    const roles = decodedToken.realm_access.roles as string[];
+
+    if (!roles.includes("admin")) {
+      history.push("/");
+    }
+
+    fetchData();
+  }, []);
+
+  /**
+   * Person Redirect click handler
+   * 
+   * @param person person data
+   */
+  const handlePersonRedirectClick = (person: PersonDto) => {
+    dispatch(setPerson(person));
+    history.push("/");
+  };
+
+  /**
+   * List Item click handler
+   * 
+   * @param event mouse event
+   * @param personWithTotalTime person with total time data
+   */
+  const handleListItemClick = (event: React.MouseEvent<HTMLDivElement, MouseEvent>, personWithTotalTime: PersonWithTotalTime) => {
+    setSelectedPersonWithTotalTime(personWithTotalTime);
+  };
+
+  /**
+   * search input change handler
+   * 
+   * @param event input change event
+   */
+  const handleSearchInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const newSearchInput = event.target.value;
+    setSearchInput(newSearchInput);
+
+    if (newSearchInput === "") {
+      setDisplayedPersonsTotalTime(personsTotalTime);
+      return;
+    }
+
+    const newDisplayedPersonsTotalTime = personsTotalTime.filter(personTotalTime =>
+      `${personTotalTime.person.firstName} ${personTotalTime.person.lastName}`.toLowerCase().includes(newSearchInput.toLowerCase()));
+
+    setDisplayedPersonsTotalTime(newDisplayedPersonsTotalTime);
+  };
+
+  /**
+   * Renders the search 
+   */
+  const renderSearch = () => {
+    return (
+      <Box className={ classes.searchContainer }>
+        <TextField
+          value={ searchInput }
+          onChange={ handleSearchInputChange }
+          placeholder={ strings.managementScreen.searchPlaceholder }
+          variant="outlined"
+          className={ classes.searchTextField }
+          inputProps={{ style: { paddingLeft: theme.spacing(5) } }}
+        />
+        <SearchIcon className={ classes.searchIcon }/>
+      </Box>
+    );
+  };
```

Comment:
Simple renders like this can use the () => (); syntax instead of () => { return ... };
```suggestion
  const renderSearch = () => (
    <Box className={ classes.searchContainer }>
      <TextField
        value={ searchInput }
        onChange={ handleSearchInputChange }
        placeholder={ strings.managementScreen.searchPlaceholder }
        variant="outlined"
        className={ classes.searchTextField }
        inputProps={{ style: { paddingLeft: theme.spacing(5) } }}
      />
      <SearchIcon className={ classes.searchIcon }/>
    </Box>
  );
```